### PR TITLE
Improve user experience for project creation

### PR DESCRIPTION
**Pull request by leewaa**
Work Package [#20884](https://community.openproject.org/work_packages/20884)

For project creation:
- [x] remove secions
  - Custom fields
  - Modules
  - Types
- [x] Add toggler 'Advanced settings' to show hide fields:
  - Subproject of
  - Identifier
  - Description
  - Project type
  - Public

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://leewaa-20884-improve-project-create.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://leewaa-20884-improve-project-create.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by leewaa** [comment]
Hey @mk84 Can you please take care of the advanced settings hide/show fields button.

**Comment by mk84** [comment]
@leewaa done and I added two new translation keys.

**Comment by leewaa** [comment]
Cheers @mk84. 

I cleaned up the cukes for this improvment. Should be green now!

File: app/views/copy_projects/copy_from_admin.html.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@ See doc/COPYRIGHT.rdoc for more details.
 <%= labelled_tabular_form_for @copy_project, url: { action: 'copy', coming_from: 'admin' } do |f| %>
   <%= hidden_field_tag :coming_from, 'admin' %>
 
-  <%= render :partial => 'projects/form', :locals => { :f => f, :project => @copy_project, :renderTypes => true } %>
+  <%= render :partial => 'projects/form', :locals => { :f => f, :project => @copy_project, :renderTypes => true, :show_all_fields => true } %>
```
Comment (oliverguenther) [inline app/views/copy_projects/copy_from_admin.html.erb:37]: No moar hash rockets!1 :grin:

**Comment by oliverguenther** [comment]
LGTM :+1: The one failing spec on travis seems unrelated and also does not fail locally.

**Pull request merged**